openshift/e2e-tests-openshift.sh

```diff
@@ -161,17 +161,9 @@ function run_e2e_tests(){
   header "Running tests"
   options=""
   (( EMIT_METRICS )) && options="-emitmetrics"
-
```

Comment:
The master branch runs the tests serially: https://github.com/openshift/knative-serving/blob/master/openshift/e2e-tests-openshift.sh#L165
If we run them in parallel the tests are more flaky because they have to wait for free resources and might time out. Running them serially helped me prevent the issue. Just FYI.